### ISSUE TITLE
RadioButton - aria-checked attribute is missing

### ISSUE THREAD

**Issue by clude86** [primefaces/primeng#17693]
### Describe the bug

When selecting a radio button, the “aria-checked” tag is not set. It is expected that the “aira-checked” is set when selecting:

`<input type="radio" class="p-radiobutton-input" value="Cheese" id="ingredient1" name="pizza" aria-checked="true" pc52="">`

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://v17.primeng.org/radiobutton

### Environment

PrimeNG v17  (17.18.15) is being used with Angular v17. The issue also replicable on https://v17.primeng.org/radiobutton

### Angular version

18.2.8

### PrimeNG version

v17 (LTS Only)

### Node version

22.5.1

### Steps to reproduce the behavior

1. Select a radio button element by clicking on label
2. Check the HTML-Code of the selected element "aria-checked = true" is excpeted

### Expected behavior

On selecting the radio button by clicking on label, the aria-checked property of the input[type="radio"] should set to "aria-checked = true"
